Assets/MixedRealityToolkit/Extensions/CameraExtensions.cs

```diff
@@ -39,5 +39,33 @@ public static bool IsInFOV(this Camera camera, Vector3 position)
 
             return (Mathf.Abs(yaw) < horizontalFovHalf && Mathf.Abs(pitch) < verticalFovHalf);
         }
+
+        /// <summary>
+        /// Gets the frustrum size at a given distance from the camera.
+        /// </summary>
+        /// <param name="camera">The camera to get the frustrum size for</param>
+        /// <param name="distanceFromCamera">The distance from the camera to get the frustrum size at</param>
+        /// <returns></returns>
+        public static Vector2 GetFrustrumSizeForDistance(this Camera camera, float distanceFromCamera)
```

Comment:
I think it's [Frustum](https://en.wikipedia.org/wiki/Frustum), not Frustrum.

Reply:
You are right! I'll fix that.